Plots/Trajectory/conda_environment.yml

```diff
@@ -0,0 +1,20 @@
+name: geocat-examples
```

Comment:
This should be removed also because of the same reasons with above two comments.